tests/system/providers/google/cloud/cloud_sql/example_cloud_sql.py

```diff
@@ -141,7 +141,7 @@
 
 with models.DAG(
     DAG_ID,
-    schedule=None,
+    schedule="@once",
```

Comment:
Not related to this PR but it's a very minor fix to other system tests that wasn't worth separate PR - "None" schedule made this tests to always pass